.github/workflows/testChecklists.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   checklist:
     runs-on: ubuntu-latest
     # Only run when a comment, PR, or PR review comment contains a checklist item
-    if: ${{ github.actor != 'OSBotify' || (github.event.issue.pull_request && github.event_name == 'issue_comment' && contains(github.event.comment.body, '- [') || github.event_name == 'pull_request_review' && contains(github.event.review.body, '- [') || github.event_name == 'pull_request' && contains(github.event.pull_request.body, '- [')) }}
+    if: github.actor != 'OSBotify' && (contains(github.event.issue.pull_request.url, 'http') && github.event_name == 'issue_comment' && contains(github.event.comment.body, '- [') || github.event_name == 'pull_request_review' && contains(github.event.review.body, '- [') || github.event_name == 'pull_request' && contains(github.event.pull_request.body, '- ['))
```

Comment:
What is the deal here? When does `pull_request` event not have an http url?

Reply:
So GitHub suggests to check `${{ github.event.issue.pull_request }}` in their docs. `pull_request` is an optional object that appears on PRs and not on issues. Checking for it in this way with additional logic doesn't seem to work as expected because it returns undefined instead of a boolean. I changed to check for `pull_request.url` containing http as that returns a boolean always.